packages/mobile/src/components/core/LinearProgress.tsx

```diff
@@ -22,10 +22,16 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
 
 type LinearProgressProps = {
   value: number
+  containerStyle?: StyleProp<ViewStyle>
+  progressStyle: StyleProp<ViewStyle>
 }
 
 export const LinearProgress = (props: LinearProgressProps) => {
-  const { value } = props
+  const {
+    value,
+    containerStyle: rootStyleProp,
```

Comment:
Let's follow the `StylesProp` pattern here https://github.com/AudiusProject/audius-client/blob/4553262d45129436095973e6e1f0a445736ff0de/packages/mobile/src/components/core/TextInput.tsx#L111